### PR TITLE
BW-1203 Initial copy paste of Terra UI setup

### PR DESCRIPTION
**Pull request by salonishah11**
Note to reviewers: Even though there are 1400+ files changes, files listed at below paths are copy paste from Terra UI. They are setup we would need later on. Please feel to not review them.

- `.editorconfig`, `.eslintignore`, `.eslintrc.js` - linting related files
- `./hooks/*` - AFAIK makes sure right yarn and node version are installed
- `./husky/*` - similar to git secrets
- `.pnp*` - yarn plugin that maps all of the packages installed in the project to where yarn has placed them on your disk
- `.yarn/*` - Terra UI commits this to repo hence I have copied it here
- `config-overrides.js` 
- `lgtm.yml` - code analysis
- `package.json` - dependencies
- `/src/fonts/*`, `/src/icons/*`, `/src/libs/*`, `src/style.css` - I thought it would be easier to have this now and be consistent with Terra UI rather than figuring out and adding it later
- `yarn.lock` - auto-generated

Closes https://broadworkbench.atlassian.net/browse/BW-1203

### CODE REVIEW

**Review by aednichols — APPROVED** [review]
Viewing the diff made my i9 laptop slow to a crawl. I say we check this in and if we discover any gremlins later we fix them in follow-on PRs.

The copy source is code that was already peer-reviewed, so how bad could it be? 😃

**Comment by kyuksel** [comment]
Why are we copying Terra UI code instead of modifying and/or adding to it?

File: src/components/HelloWorld.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    borderBottom: `2px solid ${colors.primary(0.55)}`,
+    zIndex: 2,
+    boxShadow: '3px 0 13px 0 rgba(0,0,0,0.3)'
+  }
+}
+
+export const HelloWorld = () => {
+  return div({}, [
+    div({
+      role: 'banner',
```
[the remark this answers is not in the document]
Comment (salonishah11) [reply src/components/HelloWorld.js:22]: This is the banner I added. We can remove it later on if we don't need it.

Screen Shot 2022-05-09 at 4 14 18 PM

File: src/appLoader.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import { h } from 'react-hyperscript-helpers'
+import RModal from 'react-modal'
+import Main from 'src/pages/Main'
+
+
+const appRoot = document.getElementById('root')
+
+RModal.defaultStyles = { overlay: {}, content: {} }
+RModal.setAppElement(appRoot)
+window.SATURN_VERSION = process.env.REACT_APP_VERSION
```
[the remark this answers is not in the document]
Comment (salonishah11) [reply src/appLoader.js]: I am not sure if it is needed. This is copy pasted from terra-ui. I will remove it for now.

**Comment by salonishah11** [comment]
@kyuksel this will be the new Submission service UI which will be spun up for each Cromwell app. So if we added this in Terra Ui we would have to write conditional statements everywhere that checks if this is for Cromwell app load new submission pages else load Terra UI submission pages. Also Terra UI has a lot of authentication related code which we would need to bypass as Leo will be handling authorization for app.

**Comment by salonishah11** [comment]
The conversation regarding where should the new Submission UI live has been moved to Slack. This can be merged for now as we are still in very early phase of creating the UI, and we will continue having discussions on where should this new UI live when it is time to productionize it.

**Comment by kyuksel** [comment]
For posterity, the aforementioned Slack thread regarding where the new UI should live is at https://broadinstitute.slack.com/archives/C01EHNUM73R/p1652196885532839.

**Pull request merged**